CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 6.0.0
+
+This release contains a number of major breaking changes:
+- feat: make distinct_id an optional parameter in posthog.capture and related functions
+- feat: make capture and related functions return `Optional[str]`, which is the UUID of the sent event, if it was sent
+- fix: remove `identify` (prefer `posthog.set()`), and `page` and `screen` (prefer `posthog.capture()`)
```

Comment:
What's the thinking behind removing these methods?

- I get `identify` is pretty pointless because it doesn't actually persist across captures in the same way it does for the JS Web SDK. Can't you set the user id with contexts now? I'm guess that's the better way than setting the distcint_id on every capture
- Worth keeping the `page` method so that people don't have to familiarize themselves with the `$pageview` property? Screen I'm less concerned with because it doesn't really make sense on the web and is only used in the mobile SDKs afaik